src/components/ShowView.tsx

```diff
@@ -0,0 +1,35 @@
+import "../main.css";
+import { Show } from "../Show";
+import { tagsRemover } from "../tagsRemover";
+
+interface ShowViewProps {
+  show: Show;
+  setSelectedShow: (st: string) => void;
+}
+
+export default function ShowView({
+  show,
+  setSelectedShow,
+}: ShowViewProps): JSX.Element {
+  const handleOpenEpisodes = () => {
+    setSelectedShow(show.name);
+  };
+
+  return (
+    <div className="flex-card">
+      <button className="episode-title" onClick={handleOpenEpisodes}>
+        {show.name}
+      </button>
+      <div className="body-of-episode">
```

Comment:
as this is the Show view using the word episodes here for you class name is misleading